tests/test_regrtest.py

```diff
@@ -264,7 +264,7 @@ def test_unicode_in_docstring(self) -> None:
 
         class MyClass(object):
             def method(self):
-                "With unicode : {'’'} "
+                "With unicode : {'`'} "
```

Comment:
This looks suspicious, having strange unicode characters is probably what we want to check.